src/universal/modules/teamDashboard/components/ProjectColumn/ProjectColumn.js

```diff
@@ -132,8 +126,13 @@ const ProjectColumn = (props) => {
             <ProjectCardContainer
               key={`teamCard${project.id}`}
               area={area}
-              dragProject={dragProject}
               project={project}
+              privateDragState={privateDragState}
+              ref={(c) => {
+                if (c) {
+                  privateDragState[status].components.push(c);
```

Comment:
hacky magic!
sometimes `c` returns `null`. Why it does that is beyond my knowledge of how react currently renders to the DOM.